docs/_includes/Statements.md

```diff
@@ -431,20 +485,19 @@ For this form, the guards are evaluated in some undetermined order
 until one is found that is true, in which case the corresponding statements
 are executed. If none of the guards evaluates to true, then the
 loop execution is terminated.
+k
 
-### Loop Specifications
+## Loop Specifications {#sec-loop-specification}
 For some simple loops, such as those mentioned previously, Dafny can figure
 out what the loop is doing without more help. However, in general the user
 must provide more information in order to help Dafny prove the effect of
 the loop. This information is provided by a ``LoopSpec``. A
 ``LoopSpec`` provides information about invariants, termination, and
-what the loop modifies. ``LoopSpecs`` are explained in
-section [#sec-loop-specification]. However the following sections
-present additional rationale and tutorial on loop specifications.
+what the loop modifies. 
 For additional tutorial information see [@KoenigLeino:MOD2011] or the
 [online Dafny tutorial](http://rise4fun.com/Dafny/tutorial/Guide).
 
-#### Loop Invariants
+### Loop Invariants
 
 `While` loops present a problem for Dafny. There is no way for Dafny to
```

Comment:
Again, my mindset is different. I don't think they present a problem, because the verifier knows exactly how to handle them. But one cannot reason about loops (in any language, for that matter) by considering each of its infinitely many unrollings separately--I think this is what the sentence on L502 is trying to get at.

Reply:
Reworded.